packages/ilib-loctool-webos-common/utils.js

```diff
@@ -113,4 +116,32 @@ module.exports.addNewResource = function (newresSet, res, locale) {
     newresSet.add(newres);
 
     return true;
-};
\ No newline at end of file
+};
+
+/**
+* @param {Object} settings settings object that configures how the tool will operate
+*
+* @returns {String} Information on which device is currently being targeted for localization
+*/
+module.exports.getDeviceType = function (settings) {
+    if (!settings) return;
+    return settings.metadata?.["device-type"];
```

Comment:
Can't use ?. operator in old loctool code. When we update to modern typescript, we can start using this.